docs/hosting/composer.md

```diff
@@ -78,17 +78,17 @@ the [farmOS project template](https://packagist.org/packages/farmos/project).
 You can use this `composer.json` as the starting point for your own project's
 code repository with [Git](https://git-scm.com/):
 
-    git init
-    git add composer.json
-    git commit -m 'Intitial commit of my farmOS project.'
-
-Open `composer.json` in a text editor and change the metadata (eg: `name`,
-`description`, `homepage`, etc) to customize it for your project.
+```shell
+git init
+git add composer.json
+git commit -m 'Intitial commit of my farmOS project.'
+```
 
 It is important to understand that your project is *not* farmOS, but rather
 farmOS is a *dependency* of your project. You can see this clearly by observing
 that `farmos/farmos` is included in your `composer.json` file's `require`
-section.
+section. To customize your project, open `composer.json` in a text editor and
+change the metadata (eg: `name`, `description`, `homepage`, etc).
```

Comment:
I still think there would be value in at least alluding to ***why*** it is important for readers understand this distinction. Are we just trying to avoid a bunch of confusing/misleading project.json files scattered over the Internet, or is there something that will break if they don't choose a different/distinct project name?

Reply:
I think my motivation for including this is because I want it to be clear that you are not forking/modifying farmOS itself, but rather you are creating your own project that includes farmOS as a dependency. I don't think this distinction is obvious to someone who is new to Composer. I helped some devs who are new to PHP and web dev in general set up their project and explaining this to them helped make it click.

Reply:
I think it's not uncommon for someone to think of farmOS as an "app" - but really it is an installation profile of Drupal, which must be instantiated as a dependency of a Composer project, along with scaffolding for the Drupal webroot etc. It's confusing, so I'm hoping that explicitly describing the fact that "farmOS is a dependency of *your* project" helps to clarify some of that here.

Reply:
> Are we just trying to avoid a bunch of confusing/misleading project.json files scattered over the Internet, or is there something that will break if they don't choose a different/distinct project name?

I don't think there's any risk, I just wanted to convey a better understanding of how the pieces relate.

Maybe the wording "It is **important** to understand..." is too strong? Perhaps "It is **helpful** to understand..." would be better?

Reply:
What do you think @symbioquine ? https://github.com/farmOS/farmOS/pull/648/commits/7aec2aa1b613a879689ba44ded03c3bead06c8dd

Reply:
Much better!